nano/node/active_transactions.cpp

```diff
@@ -333,12 +333,6 @@ void nano::active_transactions::request_loop ()
 	{
 		request_confirm (lock);
 		update_active_difficulty (lock);
-
-		// This prevents unnecessary waiting if stopped is set in-between the above check and now
-		if (stopped)
```

Comment:
Should it be removed?

Reply:
no it shouldn't was a missed merge commit